HOSTING.md

```diff
@@ -0,0 +1,49 @@
+## Hosting vectara-answer
+
+This guide provides specific instructions to help you host your `vectara-answer` application on one of the popular hosting providers.
+We keep adding new hosting providers regularly - please let us know if you need instructions for a hosting provider that is not yet included.
+
+### Overview
+
+To run a hosted version of `vectara-answer` you would need to have two files: `.env` and `queries.json`.
+
+To prepare the `.env` file, which combines the information from `config.yaml` and `secrets.toml`, run the following command:
+
+`python3 prepare_config.py /path/to/<YOUR-CONFIG.yaml> <profile>`
+
+At the end of this process you should have two files ready to go:
+1. `.env` generated by the above python3 command
+2. `queries.json` that specifies the queries for your application.
+
+### Deploy vectara-answer on Render
+
+To deploy on Render, please follow these steps:
+
+1. Sign Up/Log In: If you don't have a Render account, you'll need to create one. If you already have one, just log in.
+2. Create New Service: Once you're logged in, click on the "New" button usually found on the dashboard and select "Web Service".
```

Comment:
Render has some ability to do this via a configuration file, but it looks limited. It would be preferable to have this basically just do a one click method.

Reply:
Yes certainly would be better if there is a way to do this programatically but this should be a good start, as the instructions are pretty clear and simple.